brands/shop/electronics.json

```diff
@@ -1,4 +1,13 @@
 {
+   "shop/electronics|ABC_Warehouse": {
```

Comment:
Looks mostly good! - can you remove the underscore from the name here?
```suggestion
   "shop/electronics|ABC Warehouse": {
```

Reply:
@bhousel - I have removed underscore from the name. Thanks.